hw/ip/otbn/data/otbn.hjson

```diff
@@ -71,7 +71,7 @@
             The completion is signalled by the done interrupt.
           '''
           tags: [// Do not randomly start big num operations
-                 "excl:CsrNonInitTests:CsrExclWrite"],
+                 "excl:CsrAllTests:CsrExclWrite"]
```

Comment:
does "write-then-reset" work for this CSR? If not, perhaps add some comment

Reply:
This is the command field that tells OTBN to start. If we haven't loaded up a binary, we end up executing `X`s and something weird soon comes out on the interfaces, failing an `ASSERT_KNOWN` check. I'll expand the comment a bit to explain why.